msal/src/test/java/com/microsoft/identity/nativeauth/NativeAuthPublicClientApplicationJavaTest.java

```diff
@@ -2587,17 +2582,17 @@ public void testSignUpInvalidPasswordReturnsError() throws ExecutionException, I
                 MockApiResponseType.PASSWORD_TOO_WEAK
         );
 
-        SignUpUsingPasswordTestCallback signUpUsingPasswordTestCallback = new SignUpUsingPasswordTestCallback();
-        application.signUpUsingPassword(username, password, UserAttributes.Builder.customAttribute("password", password.toString()).build(), signUpUsingPasswordTestCallback);
+        SignUpTestCallback signUpTestCallback = new SignUpTestCallback();
+        application.signUp(username, password, UserAttributes.Builder.customAttribute("password", password.toString()).build(), signUpTestCallback);
```

Comment:
I never saw this before, but I find this strange. Why are we passing password as a custom attribute? It shouldn't matter technically, but we should remove it to avoid confusion.